File: crates/zoekt-rs/tests/fixtures/swift/extensions.swift
public class Person {
    public var name: String
    public init(name: String) { self.name = name }

    public func greet() {
        print("hello \(name)")
    }
}

extension Person {
    // method added via extension
    public func salute() {
        print("salute \(name)")
    }

    // static extension method
    public static func extStatic() {}
}

File: crates/zoekt-rs/tests/fixtures/rust/extensions.rs
pub struct Person {
    pub name: String,
}

impl Person {
    pub fn new(name: String) -> Self {
        Person { name }
    }
    pub fn greet(&self) {
        println!("hello {}", self.name);
    }
}

// additional impl block (extension-like)
impl Person {
    pub fn salute(&self) {
        println!("salute {}", self.name);
    }

    pub fn ext_static() {}
}
